src/index.js

```diff
@@ -0,0 +1,35 @@
+import render from './render';
+import sel from './helpers';
+import './css/style.css';
+import './css/form.css';
+
+const api = '6ca8aaad293b0b0a3ad93f29e82425f8';
+
+const fetchData = async (url) => {
+  const response = await fetch(url, { mode: 'cors' });
+  if (response.status === 401) { return console.error('BAD REQUEST | Access not allowed'); }
```

Comment:
- Let us just return instead of writing to the console. Prospective users don't worry about the message in the console, they only care about the messages displayed in the UI, kindly return the error and perhaps find a way to have them displayed in the UI as a feedback to users instead of returning and logging message at the same time based on the same error. Kindly apply this in all the instances we have the ```console``` class